src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,68 @@
+@import "../../../../public/variables";
+
+@mixin background($color,$shape) {
+  background-color: #{$color};
+  clip-path: $shape;
+  padding: $euiSizeXS - 2.5 $euiSizeL - $euiSizeXS;
+}
+
+.osdHeaderBreadcrumbs {
+  filter: url("../../public/assets/round_filter.svg#round");
+
+  .osdBreadcrumbs.first {
+    @include background($osdSanFranciscoFog,
+    polygon(
+      0% 0%,
+      100% 0%,
+      calc(100% - #{$euiSizeS}) 100%,
+      0% 100%
+    ));
+  }
+
+  .osdBreadcrumbs.middle {
+    @include background($osdSanFranciscoFog,
+    polygon(
+      $euiSizeS 0%,
+      100% 0%,
+      calc(100% - #{$euiSizeS})  100%,
+      0% 100%
+    ));
+  }
+
+  .osdBreadcrumbs.last {
+    @include background($osdPacificSky,
+    polygon(
+      $euiSizeS 0%,
+      100% 0%,
+      calc(100% - #{$euiSizeS}) 100%,
+      0% 100%
+    ));
+  }
+
+  .osdBreadcrumbs.first.last {
+    @include background($osdPacificSky,
+    polygon(
+      0% 0%,
+      100% 0%,
+      calc(100% - #{$euiSizeS})100%,
+      0% 100%
+    ));
+  }
+
+  .euiBreadcrumbSeparator {
+    display: none;
+  }
+
+  .euiBreadcrumb__collapsedLink {
+    color: $osdMidnightSky;
+    background: $euiColorEmptyShade;
+  }
+
+  .euiPopover__anchor {
+    padding: 0 $euiSizeS;
+  }
+
+  .euiBreadcrumb:not(.euiBreadcrumb--last) {
+    margin-right: 0;
+  }
```

Comment:
We dont have OUI yet

Reply:
yeah, I was talking about all the other modified styles being applied on a new class name like `.osdHeaderBreadcrumbs`, rather than targeting existing class names.

Reply:
ah, I missed the nesting